### DISCUSSION TITLE
Is the ranger authz plugin supporting file-level access control?

### DISCUSSION THREAD

**Discussion by cic1988** [apache/kyuubi#4403] [Q&A] [answered]
Hi dear community,

After checking the examples in the documentation, it does not seem to me that to restrict the file access in e.g., Azure Storage Account is possible. For example, I would like to restrict the user access to some specific files inside the data lake. Anyone has an idea? Many thanks.

Cheers.

**Comment by yaooqinn** [comment, marked as answer]
Does the file system you use support ACL？ If so, you can depend on it or use row-level filtering. For example, if users are restricted to accessing `day=2/abc.txt`, it is equivalent that there's a row filter(day!=2) for them.

**Reply by cic1988** [reply]
Thank you. The row filter is a great idea.

**Reply by cic1988** [reply]
Hello @yaooqinn . I have added you from LinkedIn. Would you please accept the invitation :) ? Thanks.

**Reply by yaooqinn** [reply]
Hi @cic1988 Can you leave your LinkedIn username here?

**Reply by cic1988** [reply]
Hi @yaooqinn sure. I have seen that the invitation was accepted :) Thanks. @at me yuangao1988.

**Reply by cic1988** [reply]
Hi @yaooqinn thank you again for the tips. I have shared you a post in LinkedIn. Would be great to have your opinion. :) looking forward to it.

**Reply by cic1988** [reply]
Hi @yaooqinn . Have replied you in LinkedIn. Any chance if you are available to have a communication? :)

**Reply by cic1988** [reply]
@yaooqinn Hi, i have listed the questions. Looking forward to your message :)

**Reply by tnk-dev** [reply]
Can you please share the mentioned linked in post :)
